.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+version: 2
+
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    reviewers:
+      - "MikuroXina"
+      - "shun-shobon"
+    target-branch: main
+    commit-message:
+      prefix: chore(npm)
```

`chore`よりは`build`が適切な気がします。[src](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type)
```suggestion
      prefix: build(deps)
```